test/minitest_ext.rb

```diff
@@ -16,7 +16,8 @@ def setup
     end
 
     def run_cmd(cmd)
-      ShopifyCli::EntryPoint.call(cmd.split(' '), @context)
+      ShopifyCli::Git.stubs(:sha).returns("bb6f42193239a248f054e5019e469bc75f3adf1b")
```

Comment:
I added this stub to fix some flakey tests I was seeing:
![image](https://user-images.githubusercontent.com/30087610/78606862-04a46c00-782c-11ea-99db-9001c5c7e9bd.png)

This would be better fixed by not calling `EntryPoint` for all tests, but that's a bigger overhaul for a future PR 😉